### PR TITLE
Update features to support 9-10 eta partition geometry.

### PR DESCRIPTION
**Pull request by geonmo**
If even chamber did not have roll 1, the chamber is skipped(roll1->roll2) by this update.

### CODE REVIEW

**Comment by dildick** [comment]
I'm blocking this one:
1) Why did you modify the try catch construction? That was the safest way to run the analyzer. If you move the analyze() part into the try catch, there is no guarantee that the analyzer will not crash in absences of a geometry.
2) I had rather seen that the LUT is constructed entirely different, in a much safer way. This is just another hack on top of a hack. 
3) Please use the most recent IBs for development. https://github.com/geonmo/cmssw/compare/cms-sw:CMSSW_6_2_X_SLHC...gem-sim-validation still shows old updates.

**Comment by geonmo** [comment]
Hello, sven.
1)
I returned code by manually. MuonGEMHits file was broken during to fix complicit problem. 
I think that MuonGEMDigis is OK.
2) 
Could you find more safe LUT which you said?

File: SLHCUpgradeSimulations/Configuration/python/gemCustoms.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def customise_DQM(process):
 def customise_Validation(process):
     process.load('Validation.Configuration.gemSimValid_cff')
     process.genvalid_all += process.gemSimValid
-
+    #process.
```
Comment (dildick) [inline SLHCUpgradeSimulations/Configuration/python/gemCustoms.py]: can you clean this up?

File: SLHCUpgradeSimulations/Configuration/python/gemCustoms.py [hunk 2]
```diff
@@ -79,7 +79,8 @@ def customise_Validation(process):
 
 def customise_harvesting(process):
     process.load('Validation.Configuration.gemPostValidation_cff')
-    process.postValidation += process.gemPostValidation
+    process.genHarvesting += process.gemPostValidation
+    #process.postValidation.remove( process.gemPostValidation )
```
Comment (dildick) [inline SLHCUpgradeSimulations/Configuration/python/gemCustoms.py]: This is still not good IMO. We should find out why it is not working in the postValidation sequence. Can you post the question on hypernews?

File: Validation/MuonGEMHits/src/GEMTrackMatch.cc [hunk 3]
```diff
@@ -58,25 +58,27 @@ void GEMTrackMatch::buildLUT()
...
-  const LocalPoint lp_top(0., top_half_striplength, 0.);
-  const LocalPoint lp_bottom(0., -bottom_half_striplength, 0.);
-  const GlobalPoint gp_top = top_chamber->toGlobal(lp_top);
-  const GlobalPoint gp_bottom = bottom_chamber->toGlobal(lp_bottom);
+  float top_half_striplength = top_chamber->specs()->specificTopology().stripLength()/2.;
+  float bottom_half_striplength = bottom_chamber->specs()->specificTopology().stripLength()/2.;
+  LocalPoint lp_top(0., top_half_striplength, 0.);
+  LocalPoint lp_bottom(0., -bottom_half_striplength, 0.);
+  GlobalPoint gp_top = top_chamber->toGlobal(lp_top);
+  GlobalPoint gp_bottom = bottom_chamber->toGlobal(lp_bottom);
```
Comment (dildick) [inline Validation/MuonGEMHits/src/GEMTrackMatch.cc:71]: why did you remove all the const's?

**Comment by dildick** [comment]
PR merged in official cmssw
